api/src/transactions.rs

```diff
@@ -1127,7 +1127,7 @@ impl TransactionsApi {
         let output = output_ext.into_transaction_output(&move_resolver);
 
         // Ensure that all known statuses return their values in the output (even if they aren't supposed to)
-        let exe_status = match status.into() {
+        let exe_status = match output.status().clone() {
```

Comment:
I think it is this line, since status is VMStatus which should contain a StatusCode, whereas output.status is more directly a statuscode. But if the output is cleaner, it is cleaner.